library_generation/generate_repo.py

```diff
@@ -118,5 +64,21 @@ def generate_from_yaml(
     )
 
 
-if __name__ == "__main__":
-    main()
+def __get_target_libraries(
+    config: GenerationConfig, target_library_names: list[str] = None
+) -> list[LibraryConfig]:
+    """
+    Returns LibraryConfig objects whose library_name is in target_library_names.
+
+    :param config: a GenerationConfig object.
+    :param target_library_names: library_name of target libraries.
+    :return:
+    """
+    if target_library_names is None:
```

Comment:
What if the list is empty? Does it mean we generate nothing or generate everything?

Reply:
> What if the list is empty

We generate nothing.

The rational is if we pass a list, then the libraries will be selected from the list. Therefore, an empty list means no generation.

If we don't pass a list, the default value is None and we generate everything.

Reply:
SG. Can you mention the behavior in the comment above? I wonder how it is going to be implemented in `generate_repo.py` later, maybe just no-op if the list is empty?

Reply:
Adde comments.